platform/ios/CHANGELOG.md

```diff
@@ -51,6 +51,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 * The error passed into `-[MGLMapViewDelegate mapViewDidFailLoadingMap:withError:]` now includes a more specific description and failure reason. ([#8418](https://github.com/mapbox/mapbox-gl-native/pull/8418))
 * Improved CPU and battery performance while animating a tilted map’s camera in an area with many labels. ([#9031](https://github.com/mapbox/mapbox-gl-native/pull/9031))
 * Fixed an issue rendering polylines that contain duplicate vertices. ([#8808](https://github.com/mapbox/mapbox-gl-native/pull/8808))
+* Fixed an issue when user starts tracking position and stops manually the animation ([#7916](https://github.com/mapbox/mapbox-gl-native/pull/7916))
```

Comment:
This should go in the “User interaction” section above. (Also, the sentence is missing a period.)